src/seedu/addressbook/AddressBook.java

```diff
@@ -486,7 +495,18 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+
+            Set<String> wordsInNameLowerCase=new HashSet<String>();
+            for (String s : wordsInName) {
+                String temp=s.toLowerCase();
+                wordsInNameLowerCase.add(temp);
+            }
+            Collection<String> keywordsLowerCase=new HashSet<String>();
+            for (String s : keywords) {
+            	String temp=s.toLowerCase();
+                keywordsLowerCase.add(temp);
+            }
```

Comment:
There is a pattern that you followed in changing the string to lower case. You can use this opportunity to abstract this into a method.